code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -377,7 +377,7 @@
 	caliber = "a762"
 	matter = list(DEFAULT_WALL_MATERIAL = 4000)
 	ammo_type = /obj/item/ammo_casing/a762
-	max_ammo = 20
+	max_ammo = 10

```

Comment:
The 10 round mag isn't labeled as a 7.62 mag, it's neither coded nor mapped in anywhere that I know of, and I'm pretty sure the Z8 Bulldog uses a 10 round magazine, so I made the 7.62 mag that actually shows up have the proper number.